packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -35,177 +52,383 @@ const MyProfile = observer(() => {
+                                                )}
+                                            </Field>
+                                            <Field name='default_advert_description'>
+                                                {({ field }) => (
+                                                    <Input
+                                                        {...field}
+                                                        type='textarea'
+                                                        label={localize('Instructions')}
+                                                        error={errors.default_advert_description}
+                                                        hint={localize('This information will be visible to everyone.')}
+                                                        className='my-profile__form-textarea'
+                                                        has_character_counter
+                                                        initial_character_count={
+                                                            my_profile_store.default_advert_description.length
+                                                        }
+                                                        max_characters={300}
+                                                    />
+                                                )}
+                                            </Field>
+                                            <div className='my-profile__footer'>
+                                                <FormError message={my_profile_store.form_error} />
+
+                                                <Button
+                                                    className={classNames('my-profile__footer-button', {
+                                                        'dc-btn--green': my_profile_store.is_submit_success,
+                                                    })}
+                                                    is_disabled={!dirty || isSubmitting || !isValid}
+                                                    is_loading={my_profile_store.is_button_loading}
+                                                    is_submit_success={my_profile_store.is_submit_success}
+                                                    text={localize('Save')}
+                                                    has_effect
+                                                    primary
+                                                    large
+                                                />
+                                            </div>
+                                        </React.Fragment>
+                                    </Form>
+                                );
+                            }}
+                        </Formik>
+                    </React.Fragment>
+                </ThemedScrollbars>
+            </DesktopWrapper>
+            <MobileWrapper>
```

Comment:
Can we please use `classNames` instead, you're currently duplicating a lot of code here.

Reply:
@aaron-binary What do you think?

Reply:
Yeah the code is almost the same, there's no need to duplicate this, instead use mixins from `devices.scss` to conditionally apply CSS rules.